ios-shim/src/iospw.h

```diff
@@ -0,0 +1,14 @@
+//
+// Created by Daniel Brotsky on 12/18/21.
+//
+
+#ifndef RUST_SECURITY_FRAMEWORK_IOSPW_H
+#define RUST_SECURITY_FRAMEWORK_IOSPW_H
+
+#include <CoreFoundation/CoreFoundation.h>
+
+extern OSStatus SecSetGenericPassword(CFStringRef service, CFStringRef account, CFDataRef password);
```

Comment:
Can you add comments why this shim is needed?

Reply:
Yep, will do.